storage/mongo/utils.js

```diff
@@ -0,0 +1,20 @@
+var MongoClient = require('mongodb').MongoClient;
+var mydb;
+
+module.exports = function (config) {
+
+  connectToServer = new Promise(function (resolve, reject) {
+    var url = "mongodb://" + config.host + ":" + config.port + "/" + config.name;
```

Comment:
Here what of in comfig file type, host, port, name any of the thongs are missing . You need to vrify if conifis presemt and whetherr it has these keys and whether those keys are `not null or undefined or an empty string`
>`not null or undefined or an empty string`
> Can be done simply like for example we need to check var x has some values but not one of above just do if(x)

if your check fails reject with err. Now as this is your custom check and there are no specific error messge as such you need to create your own error message so use
{ 'err': {'msg':'your error text here'}}